extensions/wikia/MarkWikiAsClosed/controllers/MarkWikiAsClosedController.class.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use Wikia\Logger\Loggable;
+
+class MarkWikiAsClosedController extends WikiaController {
+
+	use Loggable;
```

Comment:
The code here requires `Loggable` but does not actually use it explicitly. Is there any implicit logging? If not, then we should explicitly log 400s and 500s for troubleshooting purposes.